src/js/stores/EventStore.jsx

```diff
@@ -69,6 +69,68 @@ class EventStore extends EventEmitter {
     return this.events;
   }
 
+  updateEvent(event){
+    $.ajax({
+      url: "https://dev.calligre.com/api/event/" + event.id,
+      data : JSON.stringify(event),
+      type : 'PATCH',
+      contentType : 'application/json',
+      processData: false,
+      dataType: 'json',
+      headers: {
+        "Authorization": "Bearer " + AuthService.getToken()
+      },
+      success: function(response){
+        console.log(response);
```

Comment:
talked offline, chatted about the table randomly updating as you're updating and decided to keep lazy loading.